identity/res/values/totranslate.xml

```diff
@@ -32,4 +32,4 @@
     <string name="invalid_dob_error">Date of birth does not look valid</string>
```

Comment:
Could you move the translated strings to the default `values/string.xml` ?

Reply:
These aren’t translated yet (see [here](https://app.lokalise.com/project/747824695e51bc2f4aa912.89576472/?k=292411004) in Lokalise). We should only move them once the translation has arrived.